examples/k8s.yaml

```diff
@@ -69,16 +70,19 @@ provision:
     rm -f /etc/cni/net.d/*.conf*
     apt-get install -y kubelet kubeadm kubectl && apt-mark hold kubelet kubeadm kubectl
     systemctl enable --now kubelet
+# See <https://kubernetes.io/docs/setup/production-environment/container-runtimes/>
 - mode: system
   script: |
     #!/bin/bash
     set -eux -o pipefail
     grep SystemdCgroup /etc/containerd/config.toml && exit 0
     grep "version = 2" /etc/containerd/config.toml || exit 1
-    # Configuring a cgroup driver
+    # Configuring the systemd cgroup driver
+    # Overriding the sandbox (pause) image
     cat <<EOF >>/etc/containerd/config.toml
       [plugins]
         [plugins."io.containerd.grpc.v1.cri"]
+          sandbox_image = "$(kubeadm config images list | grep pause)"
```

Comment:
Probably safer to add `| sort -r | head -n1` ?

Reply:
You mean if there are ever **two** pause images ?

```
registry.k8s.io/kube-apiserver:v1.25.0
registry.k8s.io/kube-controller-manager:v1.25.0
registry.k8s.io/kube-scheduler:v1.25.0
registry.k8s.io/kube-proxy:v1.25.0
registry.k8s.io/pause:3.8
registry.k8s.io/etcd:3.5.4-0
registry.k8s.io/coredns/coredns:v1.9.3
```

Sure, can add that if you think that it is needed...